google-analytics-query.html

```diff
@@ -198,7 +227,10 @@
          * @attribute output
          * @type string
          */
-
+        output: {
+          type: String,
+          value: 'dataTable'
```

Comment:
`'dataTable'` is not the default here. It should just be blank, which is the same as `'json'`.